src/Components/Reports/Common/styles.js

```diff
@@ -83,7 +82,12 @@ export default StyleSheet.create({
         flexDirection: 'row'
     },
     tableRowBackground: {
-        backgroundColor: global_BorderColor_300.value  /* eslint-disable-line camelcase */
+        /*variable --pf-global--BorderColor--300 value #f0f0f0
+          also --pf-global--BackgroundColor--200
```

Comment:
I didn't get this change and the comment here. Would you mind explaining again?

Reply:
In Patternfly 5, color code for global_BorderColor_300 was different than color code provided in Patternfly 6. 
To point out this, I have added this comment.


Thank you for your comment. I reviewed this file and confirmed that it’s no longer needed, as the system export files have been removed. I’ll proceed with removing it after verifying that it isn’t referenced or used anywhere else.